src/api/routers/participantUsersRouter.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+
+import { UserController } from '../controllers/userController';
+import { enrichWithUserFromParams } from '../middleware/usersMiddleware';
+import { LoggerService } from '../services/loggerService';
+import { UserService } from '../services/userService';
+
+const createParticipantUsersRouter = () => {
+  const participantUsersRouter = express.Router({ mergeParams: true });
+  const userController = new UserController(new UserService(), new LoggerService());
```

Comment:
I thought about creating a new controller & service but I figured it probably wasn't worth it at this stage, given we may change our inversify approach for dependency injection anyway.